### DISCUSSION TITLE
sftp / scp regression from 8.6.0.0 to 8.9.1.0

### DISCUSSION THREAD

**Discussion by tdesaules** [PowerShell/Win32-OpenSSH#1997] [Q&A]
Hi guys ! 

I just move my openssh server version from 8.6.0.0 to 8.9.1.0
I can't see any issue on "pure" ssh usage but I get a "lost connection" from the client with an "unknow error" when I try to use scp or sftp

I didn't see an issue about, just to confirm before create one with more debug info.

I did the test from a Windows 10 client with the same openssh version.
The server is a Windows Server 2016 (1607 buid 14393.5356) 
I use a custom port (10022) and use ssh keys to authenticate.

have a nice day

**Comment by maertendMSFT** [comment]
Moving this to an issue thread instead of a discussion: https://github.com/PowerShell/Win32-OpenSSH/issues/2006
